src/TaskManager.Core/Models/Task.cs

```diff
@@ -18,8 +16,8 @@ public class Task
     public Guid CreateEmployeeId { get; set; }
```

Comment:
С точки зрения англа, конечно, лучше CreatedBy и employee можно убрать, если у тебя все эмплои :)
```suggestion
    public Guid CreatedById { get; set; }
```

Reply:
необязательный коммент*